inference/core/workflows/core_steps/common/vlms.py

```diff
@@ -80,3 +80,12 @@
         "description": "Model returns a JSON response with the specified fields",
     },
 }
+
+
+FLORENCE_TASKS_METADATA = {
```

Comment:
since this is relevant for this block, let's move to the block module to keep common leaner